pkg/asset/cluster/azure/azure.go

```diff
@@ -27,6 +27,7 @@ func Metadata(config *types.InstallConfig) *azure.Metadata {
 		Region:                      config.Platform.Azure.Region,
 		ResourceGroupName:           config.Azure.ResourceGroupName,
 		BaseDomainResourceGroupName: config.Azure.BaseDomainResourceGroupName,
+		ZoneName:                    config.BaseDomain,
```

Comment:
```suggestion
		BaseDomainName:                    config.BaseDomain,
```

How about this var name, to help user's understanding.